docs/_articles/en/einstein/einstein-telemetry.md

```diff
@@ -3,6 +3,6 @@ title: Telemetry and Data Collection
 lang: en
 ---
 
-Einstein for Developers is currently in beta. During beta, your feedback, prompt, and response data will be used for model training. It is assumed that you are granting Salesforce a perpetual, irrevocable, non-exclusive, worldwide, royalty-free license to use Submitted Code. See our Terms of Use for more information. 
+Einstein for Developers is currently in beta. Salesforce doesn’t collect or use your data or code to improve the Einstein for Developers product, or train a global AI model. See our Terms of Use for more information.
```

Comment:
```suggestion
See our Terms of Use for more information.
```

As per https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001jCKWpYAO/view

Reply:
Removed. Added Terms of use link.